core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1393,9 +1448,11 @@ public void run() {
 
           nameResolverBackoffPolicy = null;
           ConfigOrError configOrError = resolutionResult.getServiceConfig();
+          InternalConfigSelector resolvedConfigSelector =
+              resolutionResult.getAttributes().get(InternalConfigSelector.KEY);
```

Comment:
I don't see us making an InternalConfigSelector for the provided service config anywhere. The "normal" approach is the name resolver will not provide an InternalConfigSelector, and it isn't clear how that is supposed to be handled.

Reply:
I don't understand. What does "normal" approach do?

Reply:
If a service config provides per-method configuration, right now ServiceConfigInterceptor processes the configuration and applies it to the call via CallOptions. This is quite a different flow. I would have expected the two data flows to be more similar.

Reply:
Will be doing that in a separate PR.